pkgs/servers/monitoring/prometheus/knot-exporter.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, lib, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "knot-exporter-unstable";
+  version = "2020-01-30";
+
+  src = fetchFromGitHub {
+    owner = "ghedo";
+    repo = "knot_exporter";
+    rev = "21dd46b401e0c1aea0b173e19462cdf89e1f444e";
+    sha256 = "sha256-4au4lpaq3jcqC2JXdCcf8h+YN8Nmm4eE0kZwA+1rWlc=";
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ python3.pkgs.wrapPython ];
+  buildInputs = [ python3 ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm0755 knot_exporter $out/bin/knot_exporter
+    patchShebangs $out/bin
+    buildPythonPath ${python3.pkgs.prometheus_client}
```

Comment:
Something for another PR: prometheus_client should be renamed to prometheus-client.